tests/functional/test_functional.py

```diff
@@ -68,3 +69,22 @@ def test_nextflow(pytester):
     pytester.makefile(ext=".yml", test_nextflow=NEXTFLOW_YAML.read_text())
     result = pytester.runpytest("-v", "--keep-workflow-wd-on-fail")
     assert result.ret == 0
+
+
+@pytest.mark.functional
+def test_bash(pytester):
```

Comment:
This test does not have to be marked as a functional test with a separate entry in tox.ini. That is mostly for testing if entire workflow systems such as nextflow, snakemake, miniwdl and cromwell still work with pytest-workflow.

This is a simple test if pytest-workflow works which is not tied to any workflow system so it can be moved. 

Please add this to `test_success_messages.py` and ensure the message is as expected.